pkg/cache/v3/simple.go

```diff
@@ -606,6 +609,20 @@ func (cache *snapshotCache) Fetch(ctx context.Context, request *Request) (Respon
 	return nil, fmt.Errorf("missing snapshot for %q", nodeID)
 }
 
+// GetNodeIDs gets the all node ID in cache.
+func (cache *snapshotCache) GetNodeIDs() []string {
```

Comment:
Since we have `GetStatusKeys`, how about following the same pattern and calling this `GetSnapshotKeys`?